docs/src/pages/index.tsx

```diff
@@ -4,10 +4,29 @@
  */
 
 import Layout from "@theme/Layout";
+import Cookies from "js-cookie";
 import React from "react";
+import { v4 as uuidv4 } from "uuid";
 
+import appInsights from "@site/src/appInsights";
 import { Homepage } from "@site/src/components/home";
 
+/**
+ * Initialize Application Insights and set the user ID.
+ */
+if (typeof window !== "undefined" && appInsights) {
```

Comment:
Seems like this is enough? Especially since we don't use `window` here (`docs/src/appInsights.ts` arleady does that check and sets its export appropriately).
```suggestion
if (appInsights !== undefined) {
```